Allow namespaces to be protected and deny modification unless specified (#8444)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

# CLI flag: -ruler.max-rule-groups-per-tenant-by-namespace
[ruler_max_rule_groups_per_tenant_by_namespace: <map of string to int> | default = {}]

# (experimental) List of namespaces that are protected from modification unless

> # a special HTTP header is used. If a namespace is protected, it cannot be
> # modified via the ruler's config API only read. The value is a list of strings,

Reviewer: ```suggestion
# a special HTTP header is used. If a namespace is protected, it can only be read, not modified,
# via the ruler's configuration API. The value is a list of strings,
```

Author: Done in https://github.com/grafana/mimir/pull/8444/commits/b83ba6cd8519aa206c6af52ba4cd522813e3f6a5 - FYI for the future, these docs are auto-generated. The text to change should always be on the file named `valiation/limits.go`
